test/e2e/env/src/test/java/org/apache/shardingsphere/test/e2e/env/container/atomic/storage/DockerStorageContainer.java

```diff
@@ -76,11 +80,12 @@ protected void configure() {
             withClasspathResourceMapping(new ScenarioDataPath(scenario).getInitSQLResourcePath(Type.EXPECTED, databaseType) + "/01-expected-init.sql",
                     "/docker-entrypoint-initdb.d/01-expected-init.sql", BindMode.READ_ONLY);
         }
+        withClasspathResourceMapping("/container/init-sql/" + databaseType.getType().toLowerCase() + "/99-be-ready.sql", "/docker-entrypoint-initdb.d/99-be-ready.sql", BindMode.READ_ONLY);
```

Comment:
Why add 99-be-ready.sql?

Reply:
This is to create a 'ready' user, which is used to indicate to the code that the initialization scripts have all been executed.